contrib/kind.sh

```diff
@@ -205,307 +203,339 @@ print_params()
-  --disable-snat-multiple-gws=${OVN_DISABLE_SNAT_MULTIPLE_GWS} \
-  --ovn-empty-lb-events=${OVN_EMPTY_LB_EVENTS} \
-  --multicast-enabled=${OVN_MULTICAST_ENABLE} \
-  --k8s-apiserver=${API_URL} \
-  --ovn-master-count=${KIND_NUM_MASTER} \
-  --kind \
-  --ovn-unprivileged-mode=no \
-  --master-loglevel=${MASTER_LOG_LEVEL} \
-  --node-loglevel=${NODE_LOG_LEVEL} \
-  --dbchecker-loglevel=${DBCHECKER_LOG_LEVEL}\
-  --ovn-loglevel-northd="${OVN_LOG_LEVEL_NORTHD}" \
-  --ovn-loglevel-nb="${OVN_LOG_LEVEL_NB}" \
-  --ovn-loglevel-sb="${OVN_LOG_LEVEL_SB}" \
-  --ovn-loglevel-controller="${OVN_LOG_LEVEL_CONTROLLER}" \
-  --ovn-loglevel-nbctld="${OVN_LOG_LEVEL_NBCTLD}" \
-  --egress-ip-enable=true\
-  --v4-join-subnet=${JOIN_SUBNET_IPV4}\
-  --v6-join-subnet=${JOIN_SUBNET_IPV6}
-popd
+  kind load docker-image ${OVN_IMAGE} --name ${KIND_CLUSTER_NAME}
+}
 
-kind load docker-image ${OVN_IMAGE} --name ${KIND_CLUSTER_NAME}
+kubectl_apply_ymls() {
+  pushd ../dist/yaml
+  run_kubectl apply -f k8s.ovn.org_egressfirewalls.yaml
+  run_kubectl apply -f k8s.ovn.org_egressips.yaml
+  run_kubectl apply -f ovn-setup.yaml
+  MASTER_NODES=$(kind get nodes --name ${KIND_CLUSTER_NAME} | sort | head -n ${KIND_NUM_MASTER})
+  # We want OVN HA not Kubernetes HA
+  # leverage the kubeadm well-known label node-role.kubernetes.io/master=
+  # to choose the nodes where ovn master components will be placed
+  for n in $MASTER_NODES; do
+    kubectl label node $n k8s.ovn.org/ovnkube-db=true node-role.kubernetes.io/master="" --overwrite
+    if [ "$KIND_REMOVE_TAINT" == true ]; then
+      # do not error if it fails to remove the taint
+      kubectl taint node $n node-role.kubernetes.io/master:NoSchedule- || true
+    fi
+  done
+  if [ "$OVN_HA" == true ]; then
+    run_kubectl apply -f ovnkube-db-raft.yaml
+  else
+    run_kubectl apply -f ovnkube-db.yaml
+  fi
+  run_kubectl apply -f ovs-node.yaml
+  run_kubectl apply -f ovnkube-master.yaml
+  run_kubectl apply -f ovnkube-node.yaml
+  popd
+}
 
-pushd ../dist/yaml
-run_kubectl apply -f k8s.ovn.org_egressfirewalls.yaml
-run_kubectl apply -f k8s.ovn.org_egressips.yaml
-run_kubectl apply -f ovn-setup.yaml
-MASTER_NODES=$(kind get nodes --name ${KIND_CLUSTER_NAME} | sort | head -n ${KIND_NUM_MASTER})
-# We want OVN HA not Kubernetes HA
-# leverage the kubeadm well-known label node-role.kubernetes.io/master=
-# to choose the nodes where ovn master components will be placed
-for n in $MASTER_NODES; do
-  kubectl label node $n k8s.ovn.org/ovnkube-db=true node-role.kubernetes.io/master="" --overwrite
-  if [ "$KIND_REMOVE_TAINT" == true ]; then
-    # do not error if it fails to remove the taint
-    kubectl taint node $n node-role.kubernetes.io/master:NoSchedule- || true
+kubectl_apply_ingress() {
+  # Delete kube-proxy
+  run_kubectl -n kube-system delete ds kube-proxy
+  kind get clusters
+  kind get nodes --name ${KIND_CLUSTER_NAME}
+  kind export kubeconfig --name ${KIND_CLUSTER_NAME}
```

Comment:
this doesn't belong here, this has to run inside `kind_create_cluster` , just after the cluster has finished , after `kind create cluster ...`

Reply:
ok

Reply:
i dont think it makes sense to be in kube_delete_proxy, but its something we can change later